src/ralph/access_cards/models.py

```diff
@@ -110,3 +121,52 @@ def get_autocomplete_queryset(cls):
         return cls._default_manager.exclude(
             status=AccessCardStatus.liquidated.id
         )
+
+    @classmethod
+    @transition_action()
+    def unassign_user(cls, instances, **kwargs):
+        for instance in instances:
+            kwargs['history_kwargs'][instance.pk][
+                'affected_user'
+            ] = str(instance.user)
+            instance.user = None
+
+    @classmethod
+    @transition_action(
+        form_fields={
+            'user': {
+                'field': forms.CharField(label=_('User')),
+                'autocomplete_field': 'user',
+                'default_value': partial(autocomplete_user, field_name='user')
+            }
+        },
+    )
+    def assign_user(cls, instances, **kwargs):
+        user = get_user_model().objects.get(pk=int(kwargs['user']))
+        for instance in instances:
+            instance.user = user
+
+    @classmethod
+    @transition_action(
+        form_fields={
+            'owner': {
+                'field': forms.CharField(label=_('Owner')),
+                'autocomplete_field': 'owner',
+                'default_value': partial(autocomplete_user, field_name='owner')
+            }
+        },
+        help_text=_('text'),
```

Comment:
Perhaps, it's better to change this to something sensible.

Reply:
changed